### ISSUE TITLE
Creation of namespaces with size 100 MB failed

### ISSUE THREAD

**Issue by Greg091** [pmem/ndctl#107]
I tried to create Dax Devices with size 100MB and the second attempt caused that ndctl freeze.
After rebooting the machine, every ndctl operation caused freeze.
Commands I ran to reproduce this bug:
```
sudo ndctl create-namespace --mode devdax -s 100M -a 4k -r region0 -f
sudo ndctl create-namespace --mode devdax -s 100M -a 4k -r region0 -f
```
I know that it is a bug in the kernel (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=f1eca35a0dc7), but I think ndctl should protect users from DIMM failure.

**Comment by AlisonSchofield** [comment]
Yes, ndctl should, and eventually did, protect users from this issue where even if the first namespace starts aligned, the second namespace base is likely at +100MiB, and that is unlikely to be aligned since 100MiB is not a multiple of 16MiB, 
 
This patch, "ndctl/namespace: Validate resource alignment for dax-mode namespaces", enforces the 16MB alignment for fsdax and devdax namespaces by default.

Fixed in commit e47f490713b28c990e269fb7a4fb15495ac259ed
